test/runtime/generic-input.json

```diff
@@ -63,7 +63,8 @@
   },
   "required": [
     "house_number",
-    "array_type"
+    "array_type",
+    "houseType"
```

Comment:
As discussed on Slack, the change to the input file is required to exercise a bug in the Newtonsoft serializer. Therefore, I will keep this change in place and attempt to update the tests for the other runtimes. 

If this proves too difficult, I will create a duplicate `generic-input.json` purely for the C# test.

Reply:
I started down this path, but I realised 2 things. Firstly, I would have to install every supported runtime (there's about 10!). This would have been time-consuming, but doable. 

The harder part is that, running in Docker (to avoid installing the runtimes directly on my laptop), I was getting time-outs on each of the runtime tests (Jest is configured to timeout after 50s for most tests).

I started digging into this and there's some sort of permissions issue that wasn't immediately clear to me. I'll hopefully get all the runtimes working as part of creating the devcontainer that we've discussed separately.